client/payment-details/summary/index.tsx

```diff
@@ -223,19 +223,40 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 			<CardBody>
 				<div className="payment-details-summary">
 					<div className="payment-details-summary__section">
-						<p className="payment-details-summary__amount">
+						<p
+							className="payment-details-summary__amount"
```

Comment:
Should this class be changed / clarified since the value is different now (was shopper currency paid amount, is now merchant currency received amount).